src/cinema/domainmodel/Seat.java

```diff
@@ -0,0 +1,25 @@
+package cinema.domainmodel;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Seat {
+
+    public int getRowNumber() {
+        return rowNumber;
+    }
+
+    public int getColumnNumber() {
+        return columnNumber;
+    }
+
+    @JsonProperty("row")
+    private int rowNumber;
+
+    @JsonProperty("column")
+    private int columnNumber;
```

Comment:
Can you delete `@JsonProperty` and make it simplier?

Reply:
already deleted because of the new SeatDto class